### PR TITLE
Sidebar configuration

### LINKED ISSUE #217: Configuration for initial sidebar visibility
It would be useful to have a configuration for an initial sidebar visibility.

Thoughts of @fschmenger taken from #213:
An initial sidebar visibility option can be achieved very easily, we just have to define the config option we want for that. Be aware though, that the behavior is different for small screens (sidebar becomes modal and is initially closed), so the option might not be feasible / have the desired effect.

### PR DESCRIPTION
**Pull request by fschmenger**
This adds a `sidebar` configuration object to the application context, to customize some basic sidebar behavior - closes #217.

```JSON
"sidebar": {
  "visible": true,
  "width": 400,
  "color": "white",
  "autoScroll": true,
  "scrollDuration": 500
}
```

A new sidebar scrolling feature has been implemented, to automatically scroll to the active module, typically after a new module becomes visible.
I also decided to rework the style of the sidebar toggle button to give it a bit nicer appearance.

The documentation has been updated accordingly.

### CODE REVIEW

File: app-starter/WguAppTemplate.vue [hunk 1]
```diff
@@ -124,6 +125,14 @@
       WguEventBus.$emit('app-mounted');
     },
     methods: {
+      /**
+       * Returns the configuration object for the sidebar from app-config:
```
Comment (JakobMiksch) [inline app-starter/WguAppTemplate.vue:129]:
```suggestion
       * Returns the configuration object for the sidebar from app-config.
```

File: docs/wegue-configuration.md [hunk 2]
```diff
@@ -127,6 +128,31 @@ Below is an example for such a configuration object:
   }
 ```
 
+### sidebar
+The optional `sidebar` property customizes the behavior and layout of the application sidebar. Wegue's sidebar will be implicitly enabled, if at least one module is configured to use the sidebar as a window target, as specified by ```"win"="sidebar"``` - see the general section of the [Module Configuration](module-configuration?id=General).
```
Comment (JakobMiksch) [inline docs/wegue-configuration.md:132]:
```suggestion
The optional `sidebar` property customizes the behavior and layout of the application sidebar. Wegue's sidebar will be implicitly enabled, if at least one module is configured to use the sidebar as a window target, as specified by `"win"="sidebar"` - see the general section of the [Module Configuration](module-configuration?id=General).
```

one backtick should be enough, or am I missing something?

**Review by JakobMiksch — APPROVED** [review]
Code looks fine and it works well. 
Two minor note inline.
Otherwise ready to merge from my side.
thanks @fschmenger

**Comment by fschmenger** [comment]
OK, only a minor conflict in the documentation, was easy enough to resolve.

**Pull request merged**